README.md

```diff
@@ -50,7 +50,7 @@ print("Total size:", stats.totalSize, "bytes")
 ```
 
 ## MSLV (Minimum Supported Luau Version)
-This library requires at least Luau [0.629](https://github.com/luau-lang/luau/releases/tag/0.629) (specifically requires leading `|`, `buffer` built-in and `idiv` operator support).
+This library requires at least Luau [0.669](https://github.com/luau-lang/luau/releases/tag/0.669) (specifically requires `@self` alias, leading `|`, `buffer` built-in and `idiv` operator support).
```

Comment:
Instead of referring to an "`@self` alias", we should use the term "new require-by-string semantics", which is the official term used by Luau.